plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -229,6 +232,13 @@ def _get_target(self):
             (self.templater_selector, self.name, "target")
         )
 
+    def _get_cli_vars(self):
```

Comment:
Nitpick: could we add a type annotation for the return type (str)?

Reply:
Sure I pushed a patch